### PR TITLE
update statusline after native light dialog

### PR DESCRIPTION
**Pull request by zwim**
This change is [Reviewable](https://reviewable.io/reviews/koreader/koreader/6584)

### CODE REVIEW

**Review by Frenzie — APPROVED** [review]
Looks fine.

**Review by pazos — CHANGES_REQUESTED** [review]
To match the behaviour of the title, please broadcast the event in the android implementation of showLightDialog and *only* if the user pressed the ok button.

File: frontend/device/android/device.lua [hunk 1]
```diff
@@ -376,6 +376,9 @@ function Device:showLightDialog()
         if android.isWarmthDevice() then
```
Comment (pazos) [inline frontend/device/android/device.lua:376]: just for warmth devices? Don't you need to broadcast frontlight state change after a intensity change?.

Comment (zwim) [reply frontend/device/android/device.lua:380]: Ups. Of course!

Comment (pazos) [reply frontend/device/android/device.lua]: moved at the end of the block

**Review by pazos — APPROVED** [review]

**Pull request merged**